File: corporate-reporting/src/services/RiskRiskTypeService.ts
import { WebPartContext } from '@microsoft/sp-webpart-base';
import { EntityService } from './EntityService';
import { IDataAPI } from '../types';
import { IRiskRiskType } from '../types/RiskRiskType';

export class RiskRiskTypeService extends EntityService<IRiskRiskType> {
    constructor(spfxContext: WebPartContext, api: IDataAPI) {
        super(spfxContext, api, `/RiskRiskTypes`);
    }
}

File: ORB.Services/RiskRiskTypeService.cs
using Microsoft.AspNet.OData;
using Microsoft.EntityFrameworkCore;
using ORB.Core;
using ORB.Core.Models;
using ORB.Core.Services;
using ORB.Services.Validations;
using System;
using System.Linq;
using System.Threading.Tasks;

namespace ORB.Services
{
    public class RiskRiskTypeService : EntityService<RiskRiskType>, IEntityService<RiskRiskType>
    {
        public RiskRiskTypeService(IUnitOfWork unitOfWork) : base(unitOfWork, unitOfWork.RiskRiskTypes) { }
    }
}
